oauth.go

```diff
@@ -61,9 +61,12 @@ type AzureAuthConfig struct {
 // All fields are required unless otherwise specified
 type Config struct {
 	AzureAuthConfig
-
-	// The kms provider vault base url
-	ProviderVaultBaseURL string `json:"providerVaultBaseURL" yaml:"providerVaultBaseURL"`
+	// Location for cluster
```

Comment:
do we support both *.yaml and *.json config files? if not then let us focus on one struct tags type.